dds/DCPS/NetworkConfigMonitor.h

```diff
@@ -88,9 +100,9 @@ class OpenDDS_Dcps_Export NetworkConfigMonitor : public virtual RcObject {
   void add_address(int index, const ACE_INET_Addr& address);
```

Comment:
May need to make these public and virtual so the config updater will work correctly.